kie-wb-common-dmn/kie-wb-common-dmn-backend/src/main/java/org/kie/workbench/common/dmn/backend/definition/v1_1/DecisionTablePropertyConverter.java

```diff
@@ -95,6 +95,9 @@ public static DecisionTable wbFromDMN(final org.kie.dmn.model.api.DecisionTable
             }
             result.getOutput().add(c);
         }
+        if (result.getOutput().size() == 1) {
+            result.getOutput().get(0).setName(null); // DROOLS-3281
```

Comment:
I see comments with JIRAs numbers as a temporary thing in the code base. I would replace this line with something like:
```suggestion
            // The output name is valorized for single output decision table
            result.getOutput().get(0).setName(null);
```
I think the message above gives more context to the reader (and avoid an indirection to the JIRA).